contracts/HybridCustody.cdc

```diff
@@ -323,6 +335,10 @@ pub contract HybridCustody {
         // is not nil, any Capability returned through the `getCapability` function checks that the manager allows access first.
         access(self) var managerCapabilityFilter: Capability<&{CapabilityFilter.Filter}>?
 
+        access(self) let data: {String: AnyStruct}
+        access(self) let resources: @{String: AnyResource}
+        access(self) var display: MetadataViews.Display?
```

Comment:
What are these supposed to represent? Are we expecting a child account to only have one NFT type stored in it or any number of types? If we have any number, how is it possible to only have one display view?